solidity/random-beacon/package.json

```diff
@@ -30,7 +30,7 @@
     "prepack": "tsc -p tsconfig.export.json && hardhat export-artifacts export/artifacts"
   },
   "dependencies": {
-    "@keep-network/sortition-pools": "^2.0.0-pre.6",
+    "@keep-network/sortition-pools": "^2.0.0-pre.7",
```

Comment:
Non blocking, but shouldn't it be auto updated since we have `^`? If it's not updating then we might have a similiar issue that we had when we put a `random-beacon` dependency in `ecdsa`. cc @nkuba

Reply:
I changed it because the code requires _at least_ `2.0.0-pre.7` now, it's not going to compile with `2.0.0-pre.6`. `^2.0.0-pre.7` is reflecting it: _at least `2.0.0-pre.7`, please_.